docs/algorithms/gail.rst

```diff
@@ -1,8 +1,16 @@
+.. _gail docs:
```

Comment:
I tried to just name this `gail` instead of `gail docs`, but I think it conflicts with some references automatically generated from the code documentation

Reply:
ok